apps/haproxy/haproxy.cfg

```diff
@@ -33,14 +33,18 @@ acl host_prometheus_backend hdr_dom(host) -i prometheus.ly.fish.local
 use_backend prometheus_backend if host_prometheus_backend
 
 frontend tcp_5432_in
-mode tcp
-bind *:5432
-default_backend postgres_backend
+  bind *:5432
+  mode tcp
+  log global
+  timeout client 1440m
+  default_backend postgres_backend
 
 frontend tcp_6379_in
-mode tcp
-bind *:6379
-default_backend redis_backend
+  bind *:6379
+  mode tcp
+  log global
+  timeout client 1440m
```

Comment:
Any reason you went with 1440m? I think you can set it to 0 for infinite. That being said, we will probably want to remove the ability to connect to postgres/redis through haproxy when haproxy is used in production. Not sure about this yet, just thinking out loud. 🤔

Reply:
1440m is just a big number not to experience the issue. More about this below